### ISSUE TITLE
Is --noaur assumed for --downloadonly ?

### ISSUE THREAD

**Issue by BurhanDanger** [kitsunyan/pakku#15]
```
$ pakku -Sw st-pa<tab-completion>
warning: option --root is deprecated; use --sysroot instead
error: target not found: st-patched-git
```

**Comment by kitsunyan** [comment]
Yes, I think, since I can't download packages from AUR. What behavior do you want?

**Comment by BurhanDanger** [comment]
I'm okay if noaur is assumed in this case. pacaur used to support -Sw.

One note, pakku tab completes aur package for -Sw suggesting that it supports aur

**Comment by mrvik** [comment]
When pacman receives -Sw it downloads a package and leaves it on the pkg
cache. Maybe pakku could build and cache the package skipping the install
step

**Comment by kitsunyan** [comment]
I think implementing it as pacaur does is a good idea.
